.github/workflows/build.yml

```diff
@@ -142,18 +142,28 @@ jobs:
 
     - uses: actions/checkout@v2
 
-    - name: Cache cabal store
-      uses: actions/cache@v2
-      with:
-        path: ${{ steps.setup-haskell.outputs.cabal-store }}
-        key: cabal-store-${{ env.CACHE_VERSION }}-${{ matrix.ghc }}-${{ matrix.os }}
-
     - name: Update Hackage index
       run: cabal update
 
     - name: Cabal Configure
       run: cabal --builddir="$CABAL_BUILDDIR" configure --enable-tests
 
+    - name: Record dependencies
+      run: |
+        cabal build all --dry-run
+        cat ${{ env.PLAN_JSON }} | jq -r '."install-plan"[].id' | sort | uniq > dependencies.txt
```

Comment:
Where does `PLAN_JSON` come from?

Reply:
Good spot.  I've hard coded it to `dist-newstyle/cache/plan.json` now.